blog/_posts/2025-04-22-Enhance-security-OS-Security-Patch.md

```diff
@@ -0,0 +1,77 @@
+   - Repeats: Select the desired frequency for the maintenance window. We recommend selecting Weekly.
+   - Frequency: Select the desired day of the week for the maintenance window. We recommend selecting Sunday.
+   - Maintenance start date: Select the desired start date for the maintenance window.
+   - Maintenance start time: Select the desired start time for the maintenance window.
+   - UTC offset: Select the desired UTC offset for the maintenance window. If not set, the default is +00:00.
+   
+To learn more on configuration options via CLI visit this [page](https://learn.microsoft.com/azure/aks/auto-upgrade-node-os-image?tabs=azure-cli#set-the-node-os-autoupgrade-channel-on-a-new-cluster).
+
+## Best Practices when using OS Security Patch Channel
+   
+ Here are some tips to go about doing this. 
+
+- **Configure Maintenance Windows**: Configure [Planned Maintenance window](https://learn.microsoft.com/azure/aks/planned-maintenance?tabs=azure-cli) to apply security patches during periods of low activity. This minimizes the impact on workloads and ensures that updates are applied seamlessly. 
+- **Configure Cluster Auto-Upgrade Channel**: To maximize the benefits of OS security patches, it is recommended to enable the SecurityPatch channel alongside the [Kubernetes cluster auto upgrade channel](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster?tabs=azure-cli). This dual-channel approach ensures that both the control plane and node pools are kept up-to-date with the latest security patches.
+- **Configure Upgrade Monitoring**: Regularly monitor the status of ongoing upgrades to ensure that patches are applied successfully. Utilize tools such as the [AKS Communication Manager](https://learn.microsoft.com/azure/aks/aks-communication-manager) to get periodic notifications on OS Patching updates. 
+- **Use Release Tracker**: [AKS release tracker](https://releases.aks.azure.com/webpage/index.html) provides region by region updates on what security patch version runs in a particular region. These real-time updates are crucial for closely tracking CVEs. There is also a AKS CVE status tab.![Screenshot of the AKS release tracker showing region-specific security patch updates!](/blog/assets/images/enhance-security-with-os-security-patch/sec-patch-reltracker.jpg).
+
+
+### What Is Coming Next for OS Security Patch?
+
+The future of OS security patches promises several enhancements aimed at improving the patching process and ensuring even greater security:
+
+- [Support for Network Isolated Cluster](https://github.com/Azure/AKS/issues/4962).
+- [Snapshotting similar to Ubuntu in Azure Linux](https://github.com/Azure/AKS/issues/4963).
+- [Kubelet and ContainerD patching via OS security patch](https://github.com/Azure/AKS/issues/4964).
+- [Support for Airgapped cloud environments](https://github.com/Azure/AKS/issues/4965).
```

Comment:
Windows?

Reply:
Airgapped clouds are not public...

Reply:
Yes removed AGC and put in place windows.